tests/test_client.py

```diff
@@ -393,6 +393,18 @@ def test_client_get_get_user_charging_status_failure(authenticated_client: Charg
 
     assert exc.value.response.status_code == 500
 
+@responses.activate
+def test_client_restart_home_charger(authenticated_client: ChargePoint):
```

Comment:
Can you also add a test that triggers the failure scenario and expects the exception?

Reply:
done